src-java/base-topology/base-storm-topology/src/main/java/org/openkilda/wfm/error/MessageDispatchException.java

```diff
@@ -0,0 +1,40 @@
+/* Copyright 2022 Telstra Open Source
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License");
+ *   you may not use this file except in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ */
+
+package org.openkilda.wfm.error;
+
+import org.openkilda.messaging.MessageCookie;
+
+public class MessageDispatchException extends RuntimeException {
```

Comment:
Could you please add a note here why this and the exception bellow are made unchecked?

Reply:
You are right... I have a doubds about this decision. On the write time I do not found any advantages to force caller to handle this exception. But now,  during answering your questions about providing "root" cookie for correct error report producing I have found the advantage. So I have converted them into checked exceptions.